backend/src/v2/driver/driver.go

```diff
@@ -1645,8 +1646,8 @@ func getDAGTasks(
 			// handled with dsl.Collected.
```

Comment:
Is this comment still valid since my understanding is that this PR changes `GetExecutionsInDAG` to return unique task names if ParallelFor is used?